Sources/Purchasing/StoreKit1/StoreKitRequestFetcher.swift

```diff
@@ -27,7 +27,7 @@ class StoreKitRequestFetcher: NSObject {
 
     private let requestFactory: ReceiptRefreshRequestFactory
     private var receiptRefreshRequest: SKRequest?
-    private var receiptRefreshCompletionHandlers: [@MainActor @Sendable () -> Void]
+    private var receiptRefreshCompletionHandlers: [@Sendable () -> Void]
```

Comment:
Just to understand, we shouldn't be using `@MainActor`, right?

Reply:
exclusively in v4 of the iOS SDK. This is because it has a deployment target of iOS 11

Reply:
v5 is iOS 15+ so we can use it there